test/cfg/gnu.c

```diff
@@ -28,11 +30,10 @@
 #ifdef __gnu_linux__
 #include <error.h>
 #endif
-#ifdef _GNU_SOURCE
```

Comment:
This is not how this define works. It is to signal the header that a symbol should be made available.